### PR TITLE
Separate automation project from ui code

### PR DESCRIPTION
**Pull request by ahaessly**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +154,14 @@ For more information see: http://www.scala-sbt.org/0.13/docs/Testing.html#Test+F
 
 
 ### IntelliJ
+We now recommend opening a separate IntelliJ project for the automation test code. From 
+IntelliJ choose File->New->Project From Existing Sources and select automation/build.sbt.
+You will need to set automation as the root source. In your firecloud-ui closure project
+you may want to choose the automation directory and set it to ignore. If you add a new file
+in the scala project and choose not to add it to the git repo, you may be asked again 
+in the closure project.
```
Comment (zarsky-broad) [inline README.md]: sp: Clojure

File: README.md [hunk 2]
```diff
@@ -153,6 +154,14 @@ For more information see: http://www.scala-sbt.org/0.13/docs/Testing.html#Test+F
 
 
 ### IntelliJ
+We now recommend opening a separate IntelliJ project for the automation test code. From 
+IntelliJ choose File->New->Project From Existing Sources and select automation/build.sbt.
```
Comment (zarsky-broad) [inline README.md:158]: You might want to mention that you'll have to accept the offer from intellij to add the git root.

File: automation/project/Settings.scala [hunk 3]
```diff
@@ -15,7 +15,8 @@ object Settings {
     "-deprecation",
     "-feature",
     "-encoding", "utf8",
-    "-target:jvm-1.8"
+    "-target:jvm-1.8",
+    "-Xmax-classfile-name", "100"
```
Comment (zarsky-broad) [inline automation/project/Settings.scala]: Just checking that this was an intentional addition.

**Review by zarsky-broad — APPROVED** [review]

File: automation/project/Settings.scala (hunk 3 above)
Comment (ahaessly) [reply automation/project/Settings.scala]: this is a fix brian hasn't merged yet. good find

**Comment by ahaessly** [comment]
merging this with dev requires a change to the jenkins fiab-test job. this has been tested with the fiab-test2 job. there were failures which are currently known failures:

[info] Run completed in 14 minutes, 25 seconds.
[info] Total number of tests run: 58
[info] Suites: completed 9, aborted 0
[info] Tests: succeeded 52, failed 6, canceled 0, ignored 0, pending 0
[info] *** 6 TESTS FAILED ***
[error] Failed: Total 58, Failed 6, Errors 0, Passed 52
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.billing.BillingSpec
[error] 	org.broadinstitute.dsde.firecloud.test.analysis.MethodConfigSpec
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful

**Review by breilly2 — APPROVED** [review]
👍  As long as the tests pass (or at least don't fail because of config problems), LGTM.

**Pull request merged**